File: app/src/main/java/com/mobile/model/SeatSelected.java
package com.mobile.model;

import org.parceler.Parcel;

/**
 * Created by anubis on 6/20/17.
 */

@Parcel
public class SeatSelected {

    int row;
    int column;
    String seatName;

    public SeatSelected() {
    }

    public SeatSelected(int row, int column, String seatName) {
        this.row = row;
        this.column = column;
        this.seatName = seatName;
    }

    public int getSelectedSeatRow() {
        return row;
    }

    public int getSelectedSeatColumn() {
        return column;
    }

    public String getSeatName() {
        return seatName;
    }
}

File: app/src/main/java/com/mobile/model/SeatSelected.kt
package com.mobile.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by anubis on 6/20/17.
 */

@Parcelize
class SeatSelected(var selectedSeatRow: Int = 0,
                   var selectedSeatColumn: Int = 0,
                   var seatName:String?=null) : Parcelable